actions/views/drafts.ts

```diff
@@ -28,12 +30,32 @@ type Draft = {
 }
 
 export function getDrafts(teamId: string) {
-    return async (dispatch: DispatchFunc) => {
-        let drafts: ServerDraft[] = [];
-        drafts = await Client4.getUserDrafts(teamId);
-        const actions = (drafts || []).map((draft) => {
-            const {key, value} = transformServerDraft(draft);
-            return setGlobalItem(key, value);
+    const getLegacyDrafts = makeGetLegacyDrafts();
```

Comment:
is this something we can potentially remove once this feature was used long enough to have every draft being ported?

Reply:
If yes could we add a TODO-comment here?

Reply:
Yep you're right, I've changed the PR to use `redux-persist` to migrate existing drafts, and now `getDrafts` reconciles drafts in local storage with server drafts.